.github/workflows/ios-pr-demo-publish.yml

```
@@ -0,0 +1,477 @@
+    workflows: [iOS PR Demo Build]
+    types: [completed]
+    branches: [main]
+
+permissions:
+  actions: read
+  contents: read
+  pull-requests: read
+
+concurrency:
+  group: ios-pr-demo-publisher
+  queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)